airbyte-commons-server/src/main/java/io/airbyte/commons/server/handlers/OAuthHandler.java

```diff
@@ -168,13 +171,13 @@ public OAuthConsentRead getDestinationOAuthConsent(final DestinationOauthConsent
 
   public Map<String, Object> completeSourceOAuth(final CompleteSourceOauthRequest completeSourceOauthRequest)
```

Comment:
i think this signature can now change to package-protected with the @VisibleForTesting annotation to indicate this is should not be used outside this class except for tests.

Reply:
Makes sense, updated.